sbt-plugin/src/sbt-test/scala3/basic/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.5.8
+sbt.version=1.9.0
```

Comment:
Why do we upgrade sbt here? IIUC in sbt-test, we're sticking to the oldest version of sbt to ensure support?

The evolution seems to be:
1.5.4: https://github.com/scala-js/scala-js/commit/8ccd35cee9297d9b07f786678772c06044f54249 (1.5.x supports dotty natively)
1.5.8: https://github.com/scala-js/scala-js/commit/abc3069799354f19a82c52799b5a950fa42e5c0c (fix log4j bug)

Reply:
Ah, indeed. I blindly followed the last commit that touched `project/build.properties`, but indeed there's no reason to update in the test for this change.

Updated.